README.md

```diff
@@ -1,2 +1,307 @@
+found in the OVN architecture man page.
+
+In this post we will create a secondary network to provide east/west
+connectivity between Openshift Virtualization Virtual Machines. For that, we
+will create an overlay network (disconnected from the nodes physical network),
+and deploy a DHCP server in a pod connected to that network, which will offer
+IP addresses for the VM workloads. Keep in mind configuring IP address
+management (IPAM) in a network attachment definition for virtual machines is
+**not** supported.
+
+Finally, keep in mind this SDN approach - which uses virtual networking
+infrastructure - does not require provisioning VLANs or any other physical
+network resources.
+
+## The overlay definition
+Provision the following yaml to define the overlay which will configure the
+secondary attachment for the KubeVirt VMs.
+
+Please refer to the OVN-Kubernetes user documentation for details into each of
+the knobs.
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: flatl2
+spec:
+    config: |2
+    {
+        "cniVersion": "0.3.1",
+        "name": "flatl2",
+        "netAttachDefName": "default/flatl2",
+        "topology": "layer2",
+        "type": "ovn-k8s-cni-overlay"
+    }'
+EOF
+```
+
+The above example will configure a cluster-wide overlay without a subnet
+defined. This means the users are responsible for providing an alternative way
+to configure the VM's IP addresses for the overlay network - either manually
+configuring those via `cloud-init`, of by deploying a DHCP server in the overlay
+network. In [deploying the DHCP server](#deploying-the-dhcp-server), we show a
+simple way of achieving the latter.
+
+It is also worth to point out the value of the netAttachDefName attribute must
+match the <namespace>/<name> of the surrounding `NetworkAttachmentDefinition`
+object.
+
+## Deploying the DHCP server
+Once the network definition is provisioned, the network administrator should
+deploy the DHCP server. Use the following yaml for that:
+```bash
+cat <<EOF | kubectl apply -f -
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dhcp-server-conf
+data:
+  dhcpd.conf: |
+    authoritative;
+
+    default-lease-time 86400;
+    max-lease-time 86400;
+
+    subnet 192.168.123.0 netmask 255.255.255.0 {
+            range 192.168.123.2 192.168.123.254;
+            option broadcast-address 192.168.123.255;
+    }
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dhcp-server
+  labels:
+    app: dhcp
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dhcp
+  template:
+    metadata:
+      labels:
+        app: dhcp
+      annotations:
+        k8s.v1.cni.cncf.io/networks: '[
+          {
+            "name": "flatl2",
+            "ips": ["192.168.123.1/24"]
+          }
+        ]'
+    spec:
+      containers:
+      - name: server
+        image: quay.io/mduarted/dhcpd
```

Comment:
It might be worth to highlight how you created quay.io/mduarted/dhcpd .

Reply:
yeah, you're right. It's in the repo, but I haven't pointed to it. Thanks for pointing that out.

Reply:
Added a note about this. Is that enough ?